docs/projects/fibonacci/_posts/_posts/2019-10-04-javascript.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Fibonacci sequence in JavaScript  
+layout: default  
+last-modified: 2019-10-01 4
+featured-image:  
+tags: [javascript, fibonacci]  
+authors:[ Alina1Black ]
```

Comment:
Can you add yourself to [the author file](https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml)?

Reply:
Done